_inc/client/components/auth-iframe/index.jsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { noop } from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import {
+	isFetchingConnectUrl as _isFetchingConnectUrl,
+	getConnectUrl as _getConnectUrl,
+	fetchUserConnectionData,
+	authorizeUserInPlaceSuccess,
+	isAuthorizingUserInPlace,
+} from 'state/connection';
+
+import './style.scss';
+
+export class AuthIframe extends React.Component {
+	static displayName = 'AuthIframe';
+
+	static propTypes = {
+		title: PropTypes.string.isRequired,
+		height: PropTypes.string,
+		width: PropTypes.string,
+		scrollToIframe: PropTypes.bool,
+		onAuthorized: PropTypes.func,
+	};
+
+	static defaultProps = {
+		title: 'Link to WordPress',
```

Comment:
Should this be translatable?

Reply:
Thanks, Jeremy! I am currently working on some changes in the iframe title in the [In-place reconnect PR](https://github.com/Automattic/jetpack/pull/16475) so I'll make sure to fix that there.